expo/scheduler/views.py

```diff
@@ -440,6 +443,18 @@ def show_potential_workers(opp):
             'available_volunteers': available}
 
 
+def mail_to_user(subject, message, user):
+    send_mail(subject, message, settings.DEFAULT_FROM_EMAIL, [user.email])
+
+
+def notify_volunteer_schedule_change(profile):
+    subject = "A change has been made to your Volunteer Schedule!"
+    message = loader.get_template('scheduler/volunteer_schedule_update.tmpl')
+    c = Context({'profile': profile})
+    if not settings.DEBUG:
+        mail_to_user(subject, message.render(c), profile.user_object)
+
+
 @login_required
 def allocate_workers(request, opp_id):
     '''
```

Comment:
The same notification call needs to be made in /expo/gbe/views/volunteer_changestate_view.py - the flow for approving a volunteer application.  I think you can just add the call to notify_volunteer_schedule_change there.